index.html

```diff
@@ -5,6 +5,7 @@
         <title>Personal Portfolio</title>
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel="stylesheet" href="styles.css">
+        <link rel="stylesheet" href="style-desk.css">
```

Comment:
- [ ] [OPTIONAL] You can use `media="screen and (min-width: 768px)"` as a property to the link tag so that the referenced style file gets applied only to the specified viewport size. The whole line of code may look like 
`<link
      rel="stylesheet"
      media="screen and (min-width: 992px)"
      href="css/desktop.css"
    />`